docs/manual/resource-leak-checker.tex

```diff
@@ -50,8 +50,16 @@
 \end{Verbatim}
 
 The Resource Leak Checker supports all the command-line arguments
- listed in Section~\ref{called-methods-run-checker} for
-the Called Methods Checker.
+listed in Section~\ref{called-methods-run-checker} for
+the Called Methods Checker, plus two others:
```

Comment:
One of these is commented out, but I think this is okay as long as we merge #5173 soon after we merge this (but, the PR description should have a note about that).